### DISCUSSION TITLE
Env don't work in build

### DISCUSSION THREAD

**Discussion by Lolly1150** [vercel/next.js#36803] [Help]
Hello I tried to build my project on my server (Ubuntu) but no process.env detected how can I fix it?

**Comment by icyJoseph** [comment]
Hi,

Do you have a repo to look at? Does it work if you build it in another environment?

How is `process.env` not detected? What are you trying to do?

Do you have `.env` files, or any other way to inject the env vars?

**Reply by Lolly1150** [reply]
2

**Reply by ghost** [reply]
Can you explain what you exactly mean by "no process.env detected"?
